src/change_stream/document.rs

```diff
@@ -0,0 +1,95 @@
+use bson::{Bson, Document};
+
+/// An opaque token used for resuming an interrupted `ChangeStream`.
+///
+/// When starting a new change stream, `start_after` and `resume_after` fields on
+/// `ChangeStreamOptions` can be specified with instances of `ResumeToken`.
+///
+/// See the documentation
+/// [here](https://docs.mongodb.com/manual/changeStreams/#change-stream-resume-token) for more
+/// information on resume tokens.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct ResumeToken(pub(crate) Bson);
+
+/// A `ChangeStreamEventDocument` represents a [change event]
+/// (https://docs.mongodb.com/manual/reference/change-events/) in the associated change stream.
```

Comment:
I don't think a markdown link will work when broken across multiple comment lines like this

Reply:
I don't think they're actually split. locally they aren't. github probably just renders them in that way